frontend/src/app/main/ui/dashboard/team.cljs

```diff
@@ -571,6 +577,229 @@
+         [:div.fields-row
+          [:& fm/input {:type "text"
+                        :auto-focus? true
+                        :form form
+                        :name :uri
+                        :label (tr "modals.create-webhook.url.label")
+                        :placeholder (tr "modals.create-webhook.url.placeholder")}]]
+
+         [:div.fields-row
+          [:& fm/select {:options mtypes
+                         :label (tr "dashboard.webhooks.content-type")
+                         :default "application/json"
+                         :name :mtype}]]]
+        [:div.fields-row
+         [:div.input-checkbox.check-primary
+          [:& fm/input {:type "checkbox"
+                        :form form
+                        :name :is-active
+                        :label (tr "dashboard.webhooks.active")}]
+          ]
+         [:div.explain (tr "dashboard.webhooks.active.explain")]]]
+
+
+
+       [:div.modal-footer
+        [:div.action-buttons
+         [:input.btn-gray.btn-large
+          {:type "button"
+           :value (tr "labels.cancel")
+           :on-click #(modal/hide!)}]
+         [:& fm/submit-button
+          {:label (if webhook
+                    (tr "modals.edit-webhook")
+                    (tr "modals.create-webhook"))}]]]]]]))
+
+
+(mf/defc webhooks-hero
+  []
+  [:div.banner
+   [:div.title (tr "labels.webhooks")
+    [:div.description (tr "dashboard.webhooks.description")]]
+   [:div.create-container
+    [:div.create (tr "dashboard.webhooks.create")]]]
+
+  [:div.webhooks-hero-container
+   [:div.webhooks-hero
+    [:div.desc
+     [:h2 (tr "labels.webhooks")]
+     [:& i18n/tr-html {:label "dashboard.webhooks.description"}]]
+
+    [:div.btn-primary
+     {:on-click #(st/emit! (modal/show :webhook {}))}
+     [:span (tr "dashboard.webhooks.create")]]]])
+
+
+  (mf/defc webhook-actions
+    [{:keys [on-edit on-delete] :as props}]
+    (let [show? (mf/use-state false)]
+      [:*
+       [:span.icon {:on-click #(reset! show? true)} [i/actions]]
+       [:& dropdown {:show @show?
+                     :on-close #(reset! show? false)}
+        [:ul.dropdown.actions-dropdown
+         [:li {:on-click on-edit} (tr "labels.edit")]
+         [:li {:on-click on-delete} (tr "labels.delete")]]]]))
+
+  (mf/defc last-delivery-icon
+    [{:keys [success? text] :as props}]
+    [:div.last-delivery-icon
+     [:div.tooltip
+      [:div.label text]
+      [:div.arrow-down]]
+     (if success?
+       [:span.icon.success i/msg-success]
+       [:span.icon.failure i/msg-warning])])
+
+  (mf/defc webhook-item
+    {::mf/wrap [mf/memo]}
+    [{:keys [webhook] :as props}]
+    (let [on-edit #(st/emit! (modal/show :webhook {:webhook webhook}))
+          error-code (:error-code webhook)
+          extract-status
+          (fn [error-code]
+            (let [status (-> error-code
+                             (str/split  "-")
+                             last)]
+              (if (js/isNaN status)
```

Comment:
The satatus will never be NaN in this case, because there are no conversion to number.
Also, prefer use `parse-long` instead the js/...

Reply:
Fixed